sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1225,7 +1271,6 @@ void ExtrinsicReflectorBasedCalibrator::calibrateSensors()
   RCLCPP_INFO_STREAM(
     this->get_logger(), "Pure rotation calibration radar->lidar transform:\n"
                           << calibrated_rotation_radar_to_lidar_transformation.matrix());
-
```

Comment:
Out of curiosity. Where these blank lines deleted manually? pre-commit? another tool?

Reply:
I think I deleted it manually. Now it has the space line.